app/src/modules/content/routes/item.vue

```diff
@@ -255,20 +267,42 @@ const previewUrl = computed(() => {
 	return displayValue.value.trim() || null;
 });
 
-const { data: livePreviewMode } = useLocalStorage<'split' | 'popup'>('live-preview-mode', null);
+const livePreviewMode = useLocalStorage<'split' | 'popup'>('live-preview-mode', null);
+const livePreviewSizeStorage = useLocalStorage<number>('live-preview-size', 50);
+
+const breakpoints = useBreakpoints(breakpointsTailwind);
+const isMobile = breakpoints.smaller('sm');
```

Comment:
Let’s use `smallerOrEqual` so that it matches the CSS rule of `width > 640` and so that the breakpoint changes apply at the same width.

```suggestion
const isMobile = breakpoints.smallerOrEqual('sm');
```